### PR TITLE
#19

### PR DESCRIPTION
**Pull request by laxmankhanal**
Segue from CheklistVC to MainVC

### CODE REVIEW

File: Vaccination/Constants/ImageConstants.swift [hunk 1]
```diff
@@ -2,6 +2,9 @@ import Foundation
 import UIKit
 
 class CVImages {
+
+    static var vaccineAlreadyTaken = "ic_check_circle_yellow_48dp"
+    static var vaccineToBeTaken = "ic_check_circle_black_48dp"
```
Comment (sanjaymhj) [inline Vaccination/Constants/ImageConstants.swift]: do
static var vaccineAlreadyTaken  = UIImage(named: "......")
and use the image directly.

File: Vaccination/ViewController/VaccineCheckListVC.swift [hunk 2]
```diff
@@ -19,9 +25,9 @@ extension VaccineCheckListVC: UITableViewDataSource, UITableViewDelegate {
         checkListCell.vaccineNameLabel.text = (tempChildVaccin.vaccination!).name
         var status = VaccinationStatus.stringToStatus(withStatus: tempChildVaccin.status!)
         if status.isPending() {
-            checkListCell.vaccineStatusImage.image = UIImage(named: "ic_check_circle_black_48dp")
+            checkListCell.vaccineStatusImage.image = UIImage(named: CVImages.vaccineToBeTaken)
         } else {
-            checkListCell.vaccineStatusImage.image =  UIImage(named: "ic_check_circle_yellow_48dp")
+            checkListCell.vaccineStatusImage.image =  UIImage(named: CVImages.vaccineAlreadyTaken)
```
Comment (sanjaymhj) [inline Vaccination/ViewController/VaccineCheckListVC.swift]: see comment in CVImages

File: Vaccination/ViewController/AddChildFormTVC.swift [hunk 3]
```diff
@@ -112,6 +109,7 @@ class AddChildFormTVC: UITableViewController {
             return dateCell
 
         case 2:
+            tableView.rowHeight = 40
```
Comment (sanjaymhj) [inline Vaccination/ViewController/AddChildFormTVC.swift]: can be done from storyboard

File: Vaccination/ViewController/VaccineCheckListVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/VaccineCheckListVC.swift]: :ok:

File: Vaccination/Constants/ImageConstants.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/Constants/ImageConstants.swift]: :ok:

File: Vaccination/ViewController/AddChildFormTVC.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/AddChildFormTVC.swift]: :ok:

**Comment by sanjaymhj** [comment]
:+1:

**Pull request merged**